provision/testdata/flavor_calico-3.23.2_tar/custom_resources_aci_calico.yaml

```diff
@@ -172,4 +172,113 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: calicoctl
-  namespace: kube-system
\ No newline at end of file
+  namespace: kube-system
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: aci-containers-system
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: acc-provision-config
+  namespace: aci-containers-system
+  labels:
+    aci-containers-config-version: "dummy"
+    network-plugin: aci-containers
+data:
+  spec: |-
+    {
+        "acc_provision_input": {
+            "operator_managed_config": {
+                "enable_updates": false
+            },
+            "aci_config": {
+                "system_id": "calico-l3out-fsvi-vl",
+                "apic_hosts": [
+                    "10.30.120.100"
+                ],
+                "vrf": {
+                    "name": "mykube_vrf",
+                    "tenant": "test"
+                },
+                "cluster_l3out": {
+                    "aep": "kube-cluster",
```

Comment:
wondering if we should set the 'name" as well here: https://github.com/noironetworks/acc-provision/blob/master/provision/testdata/flavor_calico-3.23.2.inp.yaml#L8 so that it can be validated in this test